frontend/components/base/BaseModal.vue

```diff
@@ -16,10 +16,13 @@
   -->
 
 <template>
-  <div v-if="modalVisible" v-click-outside="onClickOutside" class="modal-mask">
+  <div v-if="modalVisible" :class="modalMaskClass">
```

Comment:
with modalMaskClass, you are binding with 2 classes:
 - one with a logical rendering "prevent-scroll"
 - one wich will always rendered "modal-mask"

=> instead, bind only with the class you had a logical rendering

Reply:
yeah, makes sense